packages/konsole-app/src/panels/RecordPanel.tsx

```diff
@@ -49,20 +49,19 @@ const delay = 500;
 export const RecordPanel = () => {
   const classes = useStyles();
   const registryClient = useRegistryClient();
-  const [recordType, setRecordType] = useState<string>('');
+  const [recordType, setRecordType] = useState<string>('all');
   const [recordTypes, setRecordTypes] = useState<IRecordType[]>([]);
   const [records, setRecords] = useState<IRecord[]>([]);
   const [search, setSearch] = useState('');
   const [delayedSearch, setDelayedSearch] = useState(search);
+  const recordTypesWithAll = [{ type: 'all', label: 'all' }, ...recordTypes];
 
   function refreshData() {
-    setRecordTypes([]);
-    setRecords([]);
     registryClient.getRecordTypes().then(setRecordTypes);
-    registryClient.queryRecords().then(setRecords);
+    registryClient.queryRecords(recordType === 'all' ? undefined : { type: recordType }).then(setRecords);
```

Comment:
'all' should not be used as a constant anywhere -- this is a bug. In this case fix the component so that type is undefined.

Reply:
Also please use await consistently not then.

Reply:
Ok, I will try to make the component should accept undefined, and not use 'all' for that.
Will use await.

Reply:
Done